tls/s2n_cipher_suites.c

```diff
@@ -210,6 +210,7 @@ const struct s2n_record_algorithm s2n_tls13_record_alg_chacha20_poly1305 = {
 struct s2n_cipher_suite s2n_null_cipher_suite = {
     .available = 1,
     .name = "TLS_NULL_WITH_NULL_NULL",
+    .iana_name = "TLS_NULL_WITH_NULL_NULL",
```

Comment:
Do we need both .iana_name and .name? Like, what are we printing in the script, the iana name or our name? Cause it seems like the script uses both.

Reply:
The script only uses the iana name. "name" for cipher suites is the Openssl-style name.

Is the confusion maybe that I also called the new signatures scheme field "iana_name" instead of just "name"? Would just "name" make more sense there, since there isn't any existing name field?

Reply:
Yeah, it would be easier to understand if we were only printing "name" or only printing "iana_name".

Reply:
Not everything I'm printing has an iana name. Rules and certificate types don't, and I'm not sure about kems / kem groups either. 

I suppose I could change the cipher suite "name" to "openssl_name", but I'm not sure I really see the benefit.